src/test/kotlin/com/github/jensim/megamanipulator/TestHelper.kt

```diff
@@ -0,0 +1,38 @@
+package com.github.jensim.megamanipulator
+
+import com.github.jensim.megamanipulator.settings.CodeHostSettings
+import com.github.jensim.megamanipulator.settings.ForkSetting
+import java.io.FileInputStream
+import java.io.FileNotFoundException
+import java.util.Properties
+
+object TestHelper {
```

Comment:
Iv already added func for property resolution. Might best be joined to that..?

Reply:
i'll remove the testHelper